pkg/reconciler/common/transformers.go

```diff
@@ -25,6 +25,11 @@ import (
 // transformers that are common to all components.
 func transformers(ctx context.Context, obj v1alpha1.KComponent) []mf.Transformer {
 	logger := logging.FromContext(ctx)
+
+	// TODO: Openshift Hack: Take image registry from environment instead of CR
+	imagesFromEnviron(obj)
```

Comment:
that's something we will than "patch" to all of our releases ? 

If so, looks like we should get some "create release branch" script in place :-) 

we arelady updated OWNERS and add docker image etc to the upstream relerase branch

Reply:
ack, but separately please.

Reply:
sure - just raising the "concern" - that;s all. I am not saying do all of it now :-) I can help here too ;-)

Reply:
can we name this function a bit more descriptive ? (nit, again). As it stands now it looks like a get function that ignores it return value. What's about "configureImagesFromEnviron()" ?